ot-node.js

```diff
@@ -296,6 +297,7 @@ class OTNode {
             challengeService: awilix.asClass(ChallengeService).singleton(),
             importWorkerController: awilix.asClass(ImportWorkerController).singleton(),
             importService: awilix.asClass(ImportService).singleton(),
+            permissionedDataService: awilix.asClass(PermissionedDataService).singleton(),
```

Comment:
We don't have to add this here, because on line 260 we are adding all js files that are in service folder as singleton in container